ccdproc/core.py

```diff
@@ -1575,6 +1576,163 @@ def cosmicray_median(ccd, error_image=None, thresh=5, mbox=11, gbox=0,
         raise TypeError('ccd is not an numpy.ndarray or a CCDData object.')
 
 
+def ccdmask(ratio, findbadcolumns=False, byblocks=False, ncmed=7, nlmed=7,
+            ncsig=15, nlsig=15, lsigma=9, hsigma=9, ngood=5):
+    """
+    Uses method based on the IRAF ccdmask task to generate a mask based on the
+    given input.
+
```

Comment:
Please add a statement explaining which index this function considers line and which is column. Looks like line is the first index and column is the second.